src/cli.rs

```diff
@@ -0,0 +1,37 @@
+use crate::APPLICATION_NAME;
+use clap::{Error, Parser, Subcommand};
+use std::path::PathBuf;
+
+#[derive(Subcommand, Debug)]
+pub enum Command {
+    /// Download file from cloud storage to local machine
+    Download {
+        from_path: PathBuf,
+        to_path: PathBuf,
+    },
+    /// Upload file from local machine to cloud storage
+    Upload {
+        from_path: PathBuf,
+        to_path: PathBuf,
+    },
+    /// Delete file on cloud storage
+    Delete { path: PathBuf },
+    /// List files on local machine and cloud storage
+    List,
+    /// Clear logs
+    Clear,
+}
+
+#[derive(Parser, Debug)]
+#[command(name = "cloud-storage-utilizer", version = "0.0.1", about = "Performs specified request to cloud storage", long_about = None)]
```

Comment:
It would be nice to come up with some better version sync here and with the real version, so you would not need to adjust two places with the new version.